faqs/README.md

```diff
@@ -38,10 +38,11 @@
 ### [Software](#pi-software)
 
 - What operating system (OS) does it use?
+- Can I run Windows 10 on the Raspberry Pi?
 - Updates? Upgrades? What do I do?
 - I heard about something called `rpi-update`. When should I use that?
 - The processors on the latest Raspberry Pi models are 64-bit, but I cannot find an official 64-bit OS.
-- Will it run WINE (or Windows, or other x86 software)?
+- Can I run PC software on the Raspberry Pi?
 - Will it run the Windows 8 Arm edition?
```

Comment:
I guess the 'Arm' here should be capitalised too?

Reply:
That shouldn't even be in the FAQ - nobody is trying to run Windows RT on the Pi.